cidemipyme-gatsby/src/components/sections/ServicesSection.jsx

```diff
@@ -26,23 +26,35 @@ class ServicesSection extends React.Component {
     data: PropTypes.shape(ServicesPropType).isRequired
   }
 
+  getArrow(side) {
+    return (
+      <div>
```

Comment:
why do you need a div?

Reply:
Because the div has css that makes the arrow show correctly.